src/components/home/Judges.js

```diff
@@ -1,22 +1,23 @@
 import React from 'react'
+import './stylesheet/judges.css';
 import judgesList from "../../data/judges.json"
 import GuestCard from './GuestCard'
+import { withRouter } from 'react-router-dom/cjs/react-router-dom.min';
 
 const Judges = () => {
     return (
-        <div  style = {columns}>
-            {judgesList.map((judge) => (
-                <GuestCard key={"judge_" + judge.id} member={judge} />
-            ))
-            }
+        <div id="judgesSection">
+            {/* couldn't figure out why title wasnt changing to white in css file, so did inline css */}
```

Comment:
This might be due to the name title being used elsewhere so more discript names might help with that idk

Reply:
```suggestion
        <div className="judges-section">
            <h1>OUR JUDGES</h1>
```
It would be better to do something like this, for example, instead of trying to id or class every element.  Then in the CSS file, you can target the `h1` of just the `judges-section` div like this:
```
.judges-section h1 {
    color : white;
}
```